src/store.ts

```diff
@@ -1,30 +1,99 @@
 import Vue from 'vue';
-import Vuex from 'vuex';
-import firebase from 'firebase/app';
+import Vuex, { Store } from 'vuex';
 
 import * as fb from './firebaseConfig';
+import { AppToggleableRequirementChoices } from './user-data';
+import { checkNotNull } from './utilities';
 
 Vue.use(Vuex);
 
-type State = {
-  currentUser: firebase.User | null;
+type SimplifiedFirebaseUser = { readonly displayName: string; readonly email: string };
+
+export type VuexStoreState = {
+  currentFirebaseUser: SimplifiedFirebaseUser;
```

Comment:
renamed currentUser to currentFirebaseUser to avoid confusion when we later add in AppUser.